README.md

```diff
@@ -1,161 +1,245 @@
 
-Describe between 1-3 key features of the application.
 
 
-[key_feature_1]
-[key_feature_2]
-[key_feature_3]
-(back to top)
 
-🚀 Live Demo
-Add a link to your deployed project.
 
-Live Demo Link
-(back to top)
 
-💻 Getting Started
-Describe how a new developer could make use of your project.
 
-To get a local copy up and running, follow these steps.
-
-Prerequisites
 In order to run this project you need:
+one this repository to your desired folder https://github.com/techmoves/hello-microvese.git
+
+Run the command npm install to install necessary dependencies contained in the package.json file
+
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
 
-Setup
 Clone this repository to your desired folder:
 
-Install
-Install this project with:
+<!--
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+--->
+
+### Install
+Install this project with these cmd:
+
+     1.cd Desktop/
+     2git clone using repository link
+     3.cd Hello-world/
+     4.git branch branch_name
+     5.git checkout branch_name
 
-Usage
-To run the project, execute the following command:
+gitbash and clone the repo using HTTP link
+
+<!--
+Example command:
+
+```sh
+  cd my-project
+  gem install
+```
+--->
+
+<!-- ### Usage
+
+To run the project, execute the following command: -->
+
+<!--
+Example command:
+
+```sh
+  rails server
+```
+--->
+
+### Run tests
 
-Run tests
 To run tests, run the following command:
 
-Deployment
-You can deploy this project using:
+     1.Run tests
+     2.To run tests run :
+     3.npx hint .
+     4.npx stylelint "*/.{css,scss}"
+
+<!--
+Example command:
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
 
-(back to top)
+<!-- ### Deployment
 
-👥 Authors
-Mention all of the collaborators of this project.
+You can deploy this project using: -->
 
-👤 Author1
+<!--
+Example:
 
-GitHub: @techmoves
-Twitter: @Dopezonke_
-LinkedIn: https://www.linkedin.com/in/phelelani-ndlovu-883916160/
-👤 Author2
+```sh
 
+```
+ -->
 
-(back to top)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-🔭 Future Features
-Describe 1 - 3 features you will add to the project.
+<!-- AUTHORS -->
 
- [new_feature_1]
- [new_feature_2]
- [new_feature_3]
-(back to top)
+## 👥 Authors <a name="authors"></a>
 
-🤝 Contributing
-Contributions, issues, and feature requests are welcome!
+> Mention all of the collaborators of this project.
 
-Feel free to check the issues page.
+👤 **Phelelani**
 
-(back to top)
+- GitHub: [@githubhandle](https://github.com/techmoves)
+- Twitter: [@twitterhandle](https://twitter.com/Dopezonke_)
+- LinkedIn: [LinkedIn][(https://linkedin.com/in/phelelani-ndlovu-883916160)]
 
-⭐️ Show your support
-if you find it intersting let us know we are open for collarboration
+
+
+
+
+<!-- SUPPORT -->
+
+
+
+> Write a message to encourage readers to support your project
+> to new student    from microverse they veiw this project for review
```

Comment:
- [ ] As requested by the previous reviewer please 🙏 remove comments like

> Write a message to encourage readers to support your project to new student from microverse they veiw this project for review

from your README as there purpose is to describe the use of the section from the [template](https://github.com/microverseinc/readme-template). Once you have implemented required changes you can go ahead and remove it 👍